modules/install/pages/install-uninstalling.adoc

```diff
@@ -50,7 +50,7 @@ Ubuntu and Debian::
 --
 [source,console]
 ----
-sudo dpkg -r couchbase-server
+sudo apt purge couchbase-server
```

Comment:
@ceejatec <-- Not that familiar with Linux, so is this the correct way to remove the package, given that the intro in the instructions says that all the configuration files have to be removed as well?

Reply:
Yes, that's fine - apt remove takes out an installed package, but apt purge will remove associated config files.